src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerEditorFactory.vb

```diff
@@ -84,7 +84,7 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
 
                     Dim DesignerService As IVSMDDesignerService = CType(_siteProvider.GetService(GetType(IVSMDDesignerService)), IVSMDDesignerService)
                     If DesignerService Is Nothing Then
-                        Throw New Exception(SR.GetString(SR.DFX_EditorNoDesignerService, FileName))
+                        Throw New Exception(SR.GetString(My.Resources.Microsoft_VisualStudio_AppDesigner_Designer.DFX_EditorNoDesignerService, FileName))
```

Comment:
This isn't a real example, GetString doesn't exist on the base, so the simplification rules don't prompt to change it. 

Though in saying that, I see real examples where SR.ResourceName isn't being prompted to be changed to Microsoft_VisualStudio_AppDesigner_Designer.ResourceName - that is a real bug that I'm trying to repro outside of our code base.